gcomm/src/gmcast.cpp

```diff
@@ -584,6 +584,16 @@ void gcomm::GMCast::gmcast_connect(const std::string& remote_addr)
         segment_,
         group_name_);
 
+    // TODO: Maybe there is better way ??
```

Comment:
Remote UUID is not known at this point so this looks a bit redundant on top of what is done at asio level. Updating the connection status in `handle_established()` below should be enough.